src/utils/sectionList.js

```diff
@@ -30,4 +34,6 @@ export {
   sectionListCommunity,
   sectionListDocs,
   navTutorial as sectionListTutorial,
+  navFooter as sectionListFooter,
```

Comment:
I don't think it makes sense to put these here. The other three are "section lists" because they appear in the sidebar for each type (Docs, Tutorial, and Community). I think it's best to just do

`import headerLinks from 'content/headerNav.yml`

in `Header.js`.

Reply:
> Just one nit about placement and naming.
> 
> @GasimGasimzada do you want to just have this PR be for the header/footer links and do the rest in another one so we can keep things simple?

Okay, let’s keep things simple. I will rename this PR to reflect that it is only about Footer and Header links.

Reply:
Done.